project_task_description_template/views/task_templates.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- pylint:disable=name-of-check-to-skip -->
+<odoo>
+    <record id="edit_project" model="ir.ui.view">
+        <field name="name">Task Template Field</field>
```

Comment:
**nitpick:** the name of an inherited view can be omitted or should be the same as the original one

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#inherited-xml

Reply:
Thanks, I'll fix it